src/lib/I18n.js

```diff
@@ -20,8 +20,37 @@ class I18n { // eslint-disable-line no-unused-vars
         this._fallbackLanguage = fallbackLanguage;
 
         // Re-enable next line (and remove line after) to enable internationalisation support
-        // this.language = navigator.language;
-        this.language = 'en';
+        this.language = this.detectLanguage();
+
+        window.addEventListener('focus', this._onTabFocus.bind(this));
+    }
+
+    /**
+     * @returns {string} The detected language set in the 'lang' cookie. Fallback to the browser language.
+     */
+    static detectLanguage() {
+        const cookieMatch = document.cookie.match(new RegExp('(^| )lang=([^;]+)'));
+        const cookieLang = cookieMatch && decodeURIComponent(cookieMatch[2]);
+        const browserLang = navigator.language.split('-')[0];
+
+        return cookieLang || browserLang;
```

Comment:
Calling `split` on a string does not take many CPU cycles, but it's still more work than _not_ doing it. So I suggest to only do it when actually necessary.

```suggestion

        return cookieLang || navigator.language.split('-')[0];
```